static/usage/v6/toast/layout/angular/example_component_ts.md

```diff
@@ -0,0 +1,40 @@
+```ts
+import { Component } from '@angular/core';
+import { ToastController } from '@ionic/angular';
+
+@Component({
+  selector: 'app-example',
+  templateUrl: 'example.component.html',
+})
+export class ExampleComponent {
+
+  constructor(private toastController: ToastController) {}
+  
+  async presentToast(opts) {
```

Comment:
~ Developers may receive an error/warning for no implicit any when copying this snippet into their codebase.

Can we type this to:
```ts
async presentToast(opts: ToastOptions) {
```

where `ToastOptions` comes from `@ionic/core`?